docs/tutorials/kubernetes-cluster-on-azure-with-gardener/kubernetes-cluster-on-azure-with-gardener.md

```diff
@@ -71,6 +71,12 @@ Gardener allows you to create a Kubernetes cluster on different infrastructure p
     >After completing these steps, you should see your newly created secret in the *Infrastructure Secrets* section.
 
     <img src="images/secret-stored.png">
+    
+1. Register microsoft.Network for your subscription.
```

Comment:
Thanks for adding this section. I guess there will be more Azure providers needed to create a Shoot cluster e.g. `microsoft.Compute`.

Reply:
I think we can add `compute` to this guide and we can update it later with a full list.

Reply:
can very well be the case. Maybe I should make that section a bit more generic? 
For now, this was the only error that I received when trying to create a brand new cluster.

Reply:
I think the title can be generic like `register microsoft resource providers` and replace the last step with a list where we can add all the necessary providers.
If you didn't get another error its likely due to default settings.

Reply:
pushed a change now